modules/ROOT/pages/penetration-testing-policies.adoc

```diff
@@ -17,13 +17,15 @@ against Anypoint Platform, request a Security Assessment before proceeding.
 
 To request a Security Assessment:
 
-. Contact your Salesforce Account team to request the Security Assessment Request form and Security Assessment Agreement.
+. Contact your MuleSoft Customer Success Manager to request the Security Assessment Request form and Security Assessment Agreement.
 . Complete the request form and accept the agreement.
 +
 For information required to complete the request form, see the knowledge article https://help.salesforce.com/articleView?id=000322336&type=1&mode=1[Complete the Security Assessment submission form].
-. Send your completed Security Assessment form to mailto:securityassessment@salesforce.com[securityassessment@salesforce.com].
+. Send your completed Security Assessment Request form to mailto:securityassessment@salesforce.com[securityassessment@salesforce.com].
 +
-Your account representative contacts you when your Security Assessment request is approved.
+The automated system contacts you when your Security Assessment request is approved.
```

Comment:
I'm not sure it matters... should this also be Security Assessment Request? (line 26)